src/Symfony/Component/Console/Input/InputOption.php

```diff
@@ -21,9 +21,21 @@
  */
 class InputOption
 {
+    /**
+     * Do not accept input for this option (e.g. --yell). This is the default behavior of options;
+     */
     public const VALUE_NONE = 1;
+    /**
+     * This value is required (e.g. --iterations=5 or -i5), the option itself is still optional;
+     */
     public const VALUE_REQUIRED = 2;
+    /**
+     * This option may or may not have a value (e.g. --yell or --yell=loud).
+     */
     public const VALUE_OPTIONAL = 4;
+    /**
+     * This option accepts multiple values (e.g. --dir=/foo --dir=/bar);
+     */
     public const VALUE_IS_ARRAY = 8;
     public const VALUE_NEGATABLE = 16;
```

Comment:
this constant is not documented on https://symfony.com/doc/current/console/input.html#using-command-options and I have no idea what it is supposed todo, so I left it behind.

Reply:
```suggestion
    /**
     * The option can be negated (e.g. --ansi or --no-ansi).
     */
    public const VALUE_NEGATABLE = 16;
```